packages/docs/src/routes/docs/faq/index.mdx

```diff
@@ -209,3 +210,13 @@ An amazing team of contributors around the world living in [Discord](https://qwi
 ## Is Qwik open source?
 
 Yes, [MIT](https://github.com/BuilderIO/qwik/blob/main/LICENSE) and [dependency-free](https://github.com/BuilderIO/qwik/blob/main/packages/qwik/package.json), installing Qwik will not bloat your node_modules nor your lawyers.
+
+# Does Qwik have any downside?
+
+Yes. Every framework, possessing its own strengths and weaknesses, involves a trade off.
+
+1. As a relative newcomer to the JS world, Qwik has yet to develop a community of libraries and best practices.
```

Comment:
I would change this to:

1. As a relatively new JS framework, Qwik's community and ecosystem are still under development and while growing rapidly, you might not find yet ALL the possible community projects, patterns and best practices you're used to from more popular frameworks.